### PR TITLE
[Scripts] Warning instead of error in setupvars scripts

### PR DESCRIPTION
**Pull request by akuporos**
### Details:
 - Now setupvars scripts show warning messages and don't fail if Python isn't installed or something wrong with python version.

### Tickets:
 - 94056
 - https://github.com/openvinotoolkit/openvino/issues/13411

### CODE REVIEW

File: scripts/setupvars/setupvars.bat [hunk 1]
```diff
@@ -93,14 +99,14 @@ if "%pyversion_major%" equ "%PYTHON_VERSION_MAJOR%" (
...
-   exit /B 1
+   exit /B 0
 )
 
 :: Check Python bitness
 python -c "import sys; print(64 if sys.maxsize > 2**32 else 32)" 2 > NUL
 if errorlevel 1 (
-   echo Error^: Error during installed Python bitness detection
-   exit /B 1
+   echo Warning^: Error during installed Python bitness detection
```
Comment (ilya-lavrenov) [inline scripts/setupvars/setupvars.bat:108]: for Linux we have: `64 bitness for Python $python_version is required`
So, no words like `error` which can confuse users

File: scripts/setupvars/setupvars.sh [hunk 2]
```diff
@@ -93,65 +93,80 @@ if [ -f "$INSTALLDIR/extras/opencv/setupvars.sh" ]; then
...
         fi
-    else
-        echo "[setupvars.sh] WARNING: Can not find OpenVINO Python binaries by path ${INTEL_OPENVINO_DIR}/python"
-        echo "[setupvars.sh] WARNING: OpenVINO Python environment does not set properly"
     fi
+} 
+
+python_version_to_check="$python_version"
+if [ -z "$python_version" ]; then
+        python_version_to_check="3"
```
Comment (ilya-lavrenov) [inline scripts/setupvars/setupvars.sh:161]: please, fix indentation

File: scripts/setupvars/setupvars.bat (hunk 1 above)
Comment (akuporos) [reply scripts/setupvars/setupvars.bat]: fixed

File: scripts/setupvars/setupvars.sh (hunk 2 above)
Comment (akuporos) [reply scripts/setupvars/setupvars.sh]: done

**Review by ilya-lavrenov — APPROVED** [review]

**Review by jiwaszki — APPROVED** [review]

**Review by p-wysocki — APPROVED** [review]

**Pull request merged**